### ISSUE TITLE
Infinite Save Issue

### ISSUE THREAD

**Issue by ZapperManYT** [Ajarmar/universal-pokemon-randomizer-zx#157]
At 2:30 AM I finished selecting my settings and went ahead and saved my randomized Ultra Moon file. There is an issue however, it continues to show the saving box infinitely. It still continued at 10:30 AM. After the failed attempt at the LayeredFS file, I went ahead and tried CXI, same problem. If there’s any fixes please let me know so I can finish the installation process. 

Here is my technical info.
 - Operating System: Windows 10
 - Randomizer Version: 4.1.0

**Comment by Ajarmar** [comment]
Does this happen regardless of the settings you use? What if you open up the ROM and then press Randomize immediately without changing any settings?

**Comment by ZapperManYT** [comment]
It would still have the same result regardless, because it would still save causing said error

**Comment by ZapperManYT** [comment]
I am trying the unchanged file save idea you gave and for whatever reason it still does the glitch. What is causing this? (Note: The game I'm trying to randomize is Ultra Moon, and I have a 32GB card. I didn't have an issue in past versions with this, until now.)

**Comment by tom-overton** [comment]
When you say you "didn't have an issue in past versions," can you confirm that it works in 4.0.2 for you right now? I want to make sure to remove all potential variables here. If 4.0.2 works for you right now, but 4.1.0 doesn't, then it's definitely a problem on our end I can look into.

**Comment by ZapperManYT** [comment]
I will try and experiment with 4.0.2, however I am certain the version 4.0.1 works without issue. I don't know why this is happening now though...

**Comment by tom-overton** [comment]
Yeah, that's probably too small for Ultra Moon. Try making some space before trying again. If the problem is still occurring on older versions of the randomizer, then it's probably something that has changed about your setup since the last time you tried it.

**Comment by ZapperManYT** [comment]
I cleared up some things and now my space is around 4.91 - 5 GB. I think that should work now. I'll try it and see if it works

**Comment by ZapperManYT** [comment]
Hey guys! Sorry I didn't give you any updates! As it turns out, it was just my storage space causing this "glitch". I used my Dad's desktop which has well over 400 GB, and it lagged a little bit, but after I jumped to the conclusion that it wasn't working and my randomizer was screwed, just before the PC shut down, I got a pop-up mentioning something about race mode, and I knew it worked. So yea, thanks for the support, and I'll let you guys know if there are any issues down the road!
